src/code/actions.js

```diff
@@ -268,16 +268,20 @@ function getEarnedCoinString(state) {
   return ["~ALERT.NEW_PIECE_OF_COIN", scoreString];
 }
 
+function getVisibleGenes(state, routeSpec) {
+  const visibleGenes = AuthoringUtils.getChallengeDefinition(state.authoring, routeSpec).visibleGenes;
```

Comment:
...possibly the real error was my calling the list "visibleGenes" instead of "staticGenes" or something.